cmd/rook/rook/rook.go

```diff
@@ -104,6 +106,7 @@ func GetClientset() (kubernetes.Interface, apiextensionsclient.Interface, rookcl
 	return clientset, apiExtClientset, rookClientset, nil
 }
 
+// TerminateFatal terminate the terminationlog file.
```

Comment:
a more descriptive comment could be useful, maybe:
```
// TerminateFatal terminates the process with an exit code of 1 and writes the given reason to stderr and the termination log file.
```

Reply:
Sure.